### DISCUSSION TITLE
1.16.5 - Nightly: removed statement, dancing fluids fix, /fill command fixed, block placing fixes

### DISCUSSION THREAD

**Discussion by Felicis** [Felicis/Water-Physics-Overhaul#31] [Nightly Releases]
THIS VERSION IS NIGHTLY: backup your worlds, expect crashes! Do not post issues for this, instead join the discussion :)

Hi everyone,
Finally, I've found some more time for water physics and have managed to fix another couple of bugs :partying_face:

Here's the changelog (tested only with water):

- removed statement mod dependency
- water equalization bug fixed (dancing water bug)
- fixed /fill command
- block placing animatino fix
- placing blocks has animation 
- destroying blocks in survival drops block
- no infinite blocks in survival

<hr /><em>This discussion was created from the release <a href='https://github.com/Felicis/Water-Physics-Overhaul/releases/tag/0.1.3-alpha2-hotfix'>1.16.5 - Nightly: removed statement, dancing fluids fix, /fill command fixed, block placing fixes</a>.</em>

**Comment by SnoutBug** [comment]
I think I am not bright enough to get this running;
I placed the .jar in the `mods` folder of my 1.16.5 instance with forge 36.2.39 and upon starting the game crashes with:
```
Uncaught Fatal Exception code 1
Exception in thread "main"
```
shortly after the deep scan completion of the jar.

- The provided jar is the only file in the `mods` folder.
- I am using the GD Launcher.
- I am on Fedora 37
- The instance is a clean installation

Am I missing any dependencies?

**Reply by Felicis** [reply]
I'm looking into it right now :)
For the future please post the logs (e.g. using pastebin.com). It makes it easier for me to identify the problem.

**Reply by Felicis** [reply]
I just fixed the crashes I could reproduce and updated the jar in the release. Could you try if it still crashes for you?

**Reply by SnoutBug** [reply]
It does no longer crash.

**Comment by saerdodus** [comment]
I'd like to test it at 1.18.2 for you, can you upgrade to it to 1.18.2 or do you think 1.16.5 is a good test before updating it on it

**Reply by Felicis** [reply]
I'm currently rewriting many things from scratch. I do this on 1.16.5 because it runs faster on my laptop and because it's easier to upgrade than to downgrade. I'm sometimes literally deleting my own code from days or weeks ago when completing new features, I hope you can see why I'm not upgrading it yet :|
Once the things start stabilizing and the main features are all in I'll upgrade them to 1.17 and 1.18 and then you can test them :)
